docs/search.md

```diff
@@ -184,7 +184,61 @@ var request = new SearchRepositoriesRequest("mvc client side framework")
 
 ## Search Code
 
-**TODO**
+To search for code using Octokit you need to create a `SearchCodeRequest` and populate it with the search criteria.
+
+```csharp
+// Initialize a new instance of the SearchCodeRequest class with a search term
+var request = new SearchCodeRequest("auth");
+
+// Or we can restrict the search to a specific repo
+var request = new SearchCodeRequest("auth", "dhh", "rails");
+
+var result = await githubClient.Search.SearchCode(request);
+```
+
+Now we can further filter our search.
+
+```csharp
+var request = new SearchCodeRequest("auth")
+{
+    // maybe we want to restrict the search to the file only
+    In = new[] { CodeInQualifier.File },
+
+    // or we can search the file and the path too
+    In = new[] { CodeInQualifier.File, CodeInQualifier.Path },
```

Comment:
I would prefer code that would compile. This one will fail to compile because there is duplication of `In`